tests/edl_opt_out/edl/edl_opt_out.edl

```diff
@@ -13,6 +13,7 @@ enclave {
         oe_syscall_flock_ocall;
 #ifdef OE_SGX
     from "openenclave/edl/sgx/cpu.edl" import *;
+    from "openenclave/edl/sgx/platform.edl" import *;
```

Comment:
This looks like a regression introduced by the change in enclave/sgx/attester.c. This test  explicitly checks that the attestation.edl functions do not need to be opted into by default and this PR effectively removes the weak links and forces the inclusion of  attestation.edl it seems.

Reply:
@CodeMonkeyLeet is correct. Please revert the removal of weak links in attester.c and this line as well.

Reply:
Got it now. The relevant changes will be reverted.